CMakeLists.txt

```diff
@@ -193,7 +193,7 @@ if(MSVC)
         CMAKE_CXX_FLAGS_RELEASE
         CMAKE_CXX_FLAGS_RELWITHDEBINFO
     )
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4 /MP")
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4 /MP /incremental")
```

Comment:
ping @guolinke  here

Reply:
Since it affects more than just the R package and only provides a small time savings, I think it should be removed and put in a separate PR for discussion.